test/compatibility/compatibility_test_for_releases.sh

```diff
@@ -177,6 +177,7 @@ create_configs()
 
     echo "##################################################" > $file_name
     echo "runs.type=row" >> $file_name                # WT-7379 - Temporarily disable column store tests
+    echo "block_cache=0" >> $file_name                # Disable block cache to avoid warning messages
```

Comment:
I think we need to update the comment. Looks like the pattern is to have the WT ticket when this is added (not sure why we started to do this). We should also say we forcibly disable it internally in newer releases so we don't want previous releases to enable it in compatibility testing. Finally, is there a FIXME we should add that links to the ticket that is supposed to re enable block cache? (I know we might never do this but it's still great for tracking purposes)